docs/private-networks/how-to/configure/curves.md

```diff
@@ -6,7 +6,7 @@ description: Using alternative elliptic curves in Besu
 
 !!! caution
 
-    Configuring alternative elliptic curves is an experimental feature.
+    Configuring alternative elliptic curves is an early access feature.
```

Comment:
Thanks! Everybody is using Snap and checkpoint, so I would remove the line completely
Approved! :)